utils/filesystem.py

```diff
@@ -92,9 +93,16 @@ def remove_directory_if_empty(path):
 
 
 def create_directories(path):
```

Comment:
just a thought here - python 3 has `os.makedirs(path, exist_ok=True)`. If we made this method with the same signature then it'd be really easy to switch to it when we change